src/microbe_stage/gui/PatchDetailsPanel.cs

```diff
@@ -383,31 +390,47 @@ private void UpdatePatchDetails()
             percentageFormat.FormatSafe(Math.Round(GetCompoundAmount(SelectedPatch, carbondioxideCompound),
                 Constants.ATMOSPHERIC_COMPOUND_DISPLAY_DECIMALS));
 
+        var otherAmount = CalculateOtherCompoundAmount();
+
+        otherCompoundLabel.Text = percentageFormat.FormatSafe(otherAmount,
+            Constants.ATMOSPHERIC_COMPOUND_DISPLAY_DECIMALS);
+
+        if (SelectedPatch != null)
```

Comment:
This null check is unnecessary as the calculate other compound method returns 0 if the patch is null.